src/codeflare_sdk/common/utils/constants.py

```diff
@@ -1,3 +1,4 @@
 RAY_VERSION = "2.47.1"
 # Below references ray:2.47.1-py311-cu121
 CUDA_RUNTIME_IMAGE = "quay.io/modh/ray@sha256:6d076aeb38ab3c34a6a2ef0f58dc667089aa15826fa08a73273c629333e12f1e"
+MOUNT_PATH = "/home/ray/scripts"
```

Comment:
while i'm here, just moving the new mount path to a constant since it's referenced in so many places